delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -236,8 +238,9 @@ object AclsRoutes {
 
   type AclResponseResource = ResourceF[Iri, Acl]
 
-  private val aclsRead  = Permission.unsafe("acls/read")
-  private val aclsWrite = Permission.unsafe("acls/write")
+  private val aclsRead   = Permission.unsafe("acls/read")
```

Comment:
Please do not create more permissions, but use the ones already defined in [Permissions](https://github.com/BlueBrain/nexus/blob/master/delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Permissions.scala#L130)